app/lib/models.py

```diff
@@ -0,0 +1,11 @@
+from app import db
+
+
+class Base(db.Model):
```

Comment:
Just adding another to explain the bug. Our mixin should still inherit from db.Model, and we now need to define the table name explicitly. We can do so in two ways
1) In our mixin have a function that auto does it for us: `def __tablename__(cls)`
2) On every table defined `__tablename__ = <table name>`